game_components/rooms/basic_room.py

```diff
@@ -0,0 +1,82 @@
+from abc import ABC, abstractmethod  # abstract classes
+import hashlib
+from game_components.entitiies.player import Player
+from game_components.entitiies.mob import Mob
+
+
+class BaseRoom(ABC):
+    __title = None
+    __description = None
+    __linked_rooms = None
+    __mobs = None
+    __players = None
+    __display_char = None
+    __color = None
+    uid = None
+    number = 0
+    __display_x = None
+    __display_y = None
+
+    def __init__(self, _title: str, _display_char: str, _color: (int, int, int), _description: str,
+                 _linked_rooms: dict, _display_x: int, _display_y: int):
+        """
+        Base Room
+        :param _title: Title of room
+        :param _display_char: display character of room
+        :param _color: color of tile (0-255,0-255,0-255) rgb
+        :param _description: description for room
+        :param _linked_rooms: list of linked rooms
```

Comment:
Here you ask for a list of linked rooms but the type hint calls for a `dict`

Reply:
good fix